### ISSUE TITLE
Module not found: Error: Can't resolve 'yfiles/graph-styles-other' 

### ISSUE THREAD

**Issue by kierenj** [yWorks/generator-yfiles-app#2]
When I go for ES6 modules, yarn, VS code integration, select all modules - using the latest (2.1) evaluation version, I get:

```
ERROR in ./app/scripts/app.js
Module not found: Error: Can't resolve 'yfiles/graph-styles-other' in 'C:\git\kierenj\test\app\scripts'
 @ ./app/scripts/app.js 4:0-61
 @ multi ./app/scripts/app.js
```

**Comment by yGuy** [comment]
Thanks for reporting. We found the issue, too and will fix it soon. It's a typo in the module name. It should be just "yfiles/styles-other" - please remove the "graph-" prefix and you should be fine!

**Comment by yGuy** [comment]
Fixed in 2.2.0
